src/AppBundle/Resources/public/js/app.format.js

```diff
@@ -126,7 +126,7 @@ format.info = function info(card) {
 			text += '<div>Clues: '+format.fancy_int(card.clues)+'.</div>';
 			break;
 		case 'location':
-			if (card.clues_fixed || card.clues == 0){
+			if (card.clues_fixed || !card.clues){
```

Comment:
Fixes `clues: null` formatting, which has been popping up a bit more in FHV. (in JS => `null != 0`)